docs/source/datamanagement/globus.md

```diff
@@ -65,22 +65,22 @@ After you have an account and set up a personal endpoint using Globus Connect pe
 
 ### Using the Northeastern endpoint
 
-To access the Northeastern endpoint on Globus, on the Globus web app, click **File Manager**, then in the **Collection** text box, type Northeastern. The endpoints owned by Northeastern University are displayed in the collection area. The general Northeastern endpoint is `Discovery Cluster`. This collection is a Managed Mapped Collection (GCS) hosted on dtn-03.rc.northeastern.edu. Using the File Manager interface, you can easily change directories, switch the direction of transferring to and from, and specify options such as transferring only new or changed files. Below is a procedure for transferring files from Discovery to your personal computer, but with the flexibility of the File Manager interface, you can adjust the endpoints, file view, direction of the transfer, and many other options.
+To access the Northeastern endpoint on Globus, on the Globus web app, click **File Manager**, then in the **Collection** text box, type Northeastern. The endpoints owned by Northeastern University are displayed in the collection area. The general Northeastern endpoint is `Discovery Cluster`. This collection is a Managed Mapped Collection (GCS) hosted on dtn-03.rc.northeastern.edu. Using the File Manager interface, you can easily change directories, switch the direction of transferring to and from, and specify options such as transferring only new or changed files. Below is a procedure for transferring files from Explorer to your personal computer, but with the flexibility of the File Manager interface, you can adjust the endpoints, file view, direction of the transfer, and many other options.
 
-**To transfer files from Discovery to your personal computer, do the following**
+**To transfer files from Explorer to your personal computer, do the following**
 
 1. Create an endpoint on your computer using the procedure above "Install Globus Connect," if you have not done so already.
 1. In the File Manager on the Globus web app, in the **Collections** textbox, type Discovery Cluster, then in the collection list, click the `Discovery Cluster` endpoint.
```

Comment:
text: type Discovery Cluster, then in the collection list, click the `Discovery Cluster` endpoint 
to be changed with: type Explorer Cluster, then in the collection list, click the `Explorer Cluster` endpoint

Reply:
Same as above, we'll first need to make sure the right endpoint is set up